src/internal/m365/collection/groups/channel_handler.go

```diff
@@ -67,6 +67,15 @@ func (bh channelsBackupHandler) canonicalPath(
 			false)
 }
 
+func (bh channelsBackupHandler) MessagePathPrefix(tenantID string) (path.Path, error) {
```

Comment:
`message` won't abstract well if/when we add other handlers.
```suggestion
func (bh channelsBackupHandler) PathPrefix(tenantID string) (path.Path, error) {
```